examples/foundational/13b-deepgram-transcription.py

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (c) 2024, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+import asyncio
+import os
+import sys
+
+from pipecat.frames.frames import Frame, TranscriptionFrame
+from pipecat.pipeline.pipeline import Pipeline
+from pipecat.pipeline.runner import PipelineRunner
+from pipecat.pipeline.task import PipelineTask
+from pipecat.processors.frame_processor import FrameDirection, FrameProcessor
+from pipecat.services.deepgram import DeepgramSTTService
+from pipecat.transports.services.daily import DailyParams, DailyTransport
+
+from runner import configure
+
+from loguru import logger
+
+from dotenv import load_dotenv
+load_dotenv(override=True)
+
+logger.remove(0)
+logger.add(sys.stderr, level="DEBUG")
+
+
+class TranscriptionLogger(FrameProcessor):
+
+    async def process_frame(self, frame: Frame, direction: FrameDirection):
+        await super().process_frame(frame, direction)
```

Comment:
Can you add a comment to explain why we need to call `super().process frame` here? It isn't immediately obvious to me, so it would probably help others too

Reply:
Yeah, this is a new thing. It's strictly not necessary at all, but I added it everywhere as a best practice. The reason is that, for example, `StartFrame` comes with data (e.g. `allow_interruptions` or `enable_metrics`) and to avoid for each `FrameProcessor` to handle that we now handle it in the parent class. So, it's now a best practice to do that.

Reply:
So, we should document this.